src/PointwiseFunctions/GeneralRelativity/GeneralizedHarmonic/Christoffel.cpp

```diff
@@ -0,0 +1,78 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "PointwiseFunctions/GeneralRelativity/GeneralizedHarmonic/Christoffel.hpp"
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Utilities/ContainerHelpers.hpp"
+#include "Utilities/GenerateInstantiations.hpp"
+#include "Utilities/Gsl.hpp"
+
+namespace GeneralizedHarmonic {
+template <size_t SpatialDim, typename Frame, typename DataType>
+void christoffel_second_kind(
+    const gsl::not_null<tnsr::Ijj<DataType, SpatialDim, Frame>*> christoffel,
+    const tnsr::iaa<DataType, SpatialDim, Frame>& phi,
+    const tnsr::II<DataType, SpatialDim, Frame>& inv_metric) noexcept {
+  if (UNLIKELY(get_size(get<0, 0, 0>(*christoffel)) !=
+               get_size(get<0, 0, 0>(phi)))) {
+    *christoffel =
+        tnsr::Ijj<DataType, SpatialDim, Frame>(get_size(get<0, 0, 0>(phi)));
```

Comment:
(not a change request) I often wonder when to do this and when `make_with_value`. Do you have a heuristic of which to use when? I've asked this before, but the answer never seems to stick...maybe this time it will?

Reply:
Good question.  I changed the `make_with_value` in christoffel_second_kind to an explicit construction, in both the gr and gh versions.  My reasoning is that I don't need to waste cycles filling it with a value at all, because I am overwriting it anyway.